htdocs/include/common.php

```diff
@@ -38,8 +38,12 @@
 /** make sure mainfile is included, for security and functionality */
 defined("XOOPS_MAINFILE_INCLUDED") or die();
 
-/** @todo This funtion has been deprecated in PHP 5.3.0 - need to adjust in 1.4 ~skenow */
-@set_magic_quotes_runtime(0);
+/** @todo when this is no longer possible to run under PHP 5.x, we can remove the check - fiammybe 12 may 2019 */
```

Comment:
I think we can remove that. PHP 5.x are now unsupported versions. We should not care about this anymore.